downloader/src/main/kotlin/Downloader.kt

```diff
@@ -97,8 +98,8 @@ object Downloader {
         val originalVcsInfo: VcsInfo? = null
     ) {
         init {
-            require((sourceArtifact == null) != (vcsInfo == null)) {
-                "Either sourceArtifact or vcsInfo must be set, but not both."
+            require(sourceArtifact == null || vcsInfo == null) {
+                "Either sourceArtifact or vcsInfo or both must be null."
```

Comment:
Maybe better "Not both sourceArtifact and vcsInfo may be set."

Reply:
Done.